backend/src/index.ts

```diff
@@ -14,123 +19,146 @@ app.use(cors());
-                await db.insert(unmatchedTable).values(query[0]);
-            }
-        }
-        else {
-            throw new Error("ID not found in unmatched table");
-        }
-    }
-    else {
-        throw new Error("Invalid ID");
+  if (id.length == 7 || id.length == 8) {
+    const query = await db
+      .select()
+      .from(unmatchedTable)
+      .where(eq(unmatchedTable.id, id)); //returns an array with only one element
+    const remain = query.length - 1;
+    if (query.length > 0) {
+      await db.insert(matchedTable).values(query[0]); //inserts only one row into matchedTable
+
+      await db.delete(unmatchedTable).where(eq(unmatchedTable.id, id));
+
+      for (let i = 0; i < query.length - 1; i++) {
+        //adding rows back in
+        await db.insert(unmatchedTable).values(query[0]);
+      }
+    } else {
+      throw new Error("ID not found in unmatched table");
     }
+  } else {
+    throw new Error("Invalid ID");
+  }
 }
 
 // moveToMatched("1000002");
 
 /******* Move a given tutor/tutee from matched to unmatched *******
- * 
+ *
  *  - given a tutor or tutee's id, move them from the matched table to the
  *    unmatched table
- * 
+ *
  *  - throw an error if they don't exist in the matched table
- * 
+ *
  *  - it is ok if the tables have duplicate ids, you just have to move 1.
- * 
+ *
  ******************************************************************/
 async function moveToUnmatched(id: string) {
-    if (id.length == 7) {
-        const query = await db.select().from(matchedTable).where(eq(matchedTable.id, id));
-        console.log(query);
-
-        if(!query) {
-            throw new Error('${id} does not exist in matched table');
-        }
-
-        await db.insert(unmatchedTable).values(query);
-        await db.delete(matchedTable).where(eq(matchedTable.id, id));
+  if (id.length == 7) {
+    const query = await db
+      .select()
+      .from(matchedTable)
+      .where(eq(matchedTable.id, id));
+    console.log(query);
+
+    if (!query) {
+      throw new Error("${id} does not exist in matched table");
     }
+
+    await db.insert(unmatchedTable).values(query);
+    await db.delete(matchedTable).where(eq(matchedTable.id, id));
+  }
 }
 
 /************** Make a match given tutor and tutee ids ***************
- * 
- *  - Given a tutor id and a tutee id, find the ids in the tutor and tutee 
+ *
+ *  - Given a tutor id and a tutee id, find the ids in the tutor and tutee
  *    unmatched tables and move them to matched
- * 
+ *
  *  - Create a new row in the approved_matched table that contains the
  *    tutor and tutee ids
- * 
+ *
  *  - Throw an error if either the tutor or tutee don't exist in the unmatched
  *    table
  */
-async function createMatch(tutor_id: string, tutee_id: string) {}
\ No newline at end of file
+async function createMatch(tutor_id: string, tutee_id: string) {}
+
+async function fetchAllTutees() {
```

Comment:
Adding this function specifically for the selection of all Tutees is fine but since it is such a simple query, I am just going to add it in-line with the endpoint.